code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Medicine.dm

```diff
@@ -716,17 +735,27 @@
 			if(prob(33))
 				H.Confuse(10)
 
+/datum/reagent/nanoperidaxon/overdose(var/mob/living/carbon/M, var/alien, var/removed) //Oversoing on Nanoperidaxon greatly reduces its organ healing effects
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		for(var/obj/item/organ/I in H.internal_organs)
+			if(I.damage > 0)
+				I.damage = max(I.damage + 0.75*removed, 0) //When overdosed Peri heals at 1/4th its normal rate
+
 /datum/reagent/osteodaxon
 	name = "Osteodaxon"
 	id = "osteodaxon"
-	description = "An experimental drug used to heal bone fractures."
+	description = "A slightly toxic experimental drug used to heal bone fractures. Requires a signifcant dose to be effective."
 	reagent_state = REAGENT_LIQUID
 	color = "#C9BCE3"
 	metabolism = REM * 0.5
-	overdose = REAGENTS_OVERDOSE * 0.5
+	overdose = 9 //This is so Bone Healing Hypos Can heal Osteo based on the rework. Trade easily treatable toxin for a trip to the surgery table.
```

Comment:
making the threshold for this "9" just seems like purposefully making it a pain in the dick because most reagent solutions in the game work on increments of 5.